### PR TITLE
fix: at the end of make-dist.sh, copy wrong file.

### PR DESCRIPTION
**Pull request by i8run**
## What changes were proposed in this pull request?

copy right jar and zip, which has been changed with SNAPSHOT.

## How was this patch tested?

manual.

### CODE REVIEW

File: make-dist.sh [hunk 1]
```diff
@@ -71,6 +71,6 @@ cp $BASEDIR/scripts/classes.lst $BIN_DIR/
 cp $BASEDIR/scripts/img_class.lst $BIN_DIR/
 cp $BASEDIR/spark/dl/src/main/resources/spark-bigdl.conf $CONF_DIR/
 
-cp $BASEDIR/spark/dl/target/bigdl-$VERSION-jar-with-dependencies.jar $LIB_DIR/
```
Comment (srowen) [inline make-dist.sh]: It generally won't be a SNAPSHOT build, right?

Comment (yiheng) [inline make-dist.sh]: On master, it's a SNAPSHOT build. On release branch(e.g. branch-0.1), it's not SNAPSHOT build.

**Comment by yiheng** [comment]
I think we needn't this if PR#768 is merge.

**Comment by yiheng** [comment]
PR#768 is merge.

**Pull request closed without merging**